lib/payment_calculator/ecf/service_fees_for_band.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "payment_calculator/ecf/contract/service_fee_calculations"
+
+module PaymentCalculator
+  module Ecf
+    class ServiceFeesForBand
+      include Ecf::Contract::ServiceFeeCalculations
+
+      def call(band:)
+        {
+          service_fee_monthly: service_fee_monthly(band).round(0),
+          service_fee_per_participant: service_fee_per_participant(band).round(0),
+          service_fee_total: service_fee_total(band).round(0),
```

Comment:
`round(0)`? Are we sure about that? I think I saw some `round(2)`s floating around...

Reply:
The requirement is to round to the nearest pound